README.md

```diff
@@ -58,6 +62,28 @@ If you cannot control how your program is run, you can load the `newrelic` modul
     /* ... the rest of your program ... */
 ```
 
+## ES Modules
```

Comment:
I'd expand the acronym, ECMAScript Modules.

Reply:
blah. ok just this once 😄